db/migrate/20210806113022_create_restaurants_table.rb

```diff
@@ -0,0 +1,13 @@
+class CreateRestaurantsTable < ActiveRecord::Migration[6.1]
+  def change
+    create_table :restaurants do |t|
+      t.string :name, null: false, default: ""
+      t.text :phone_number
```

Comment:
just change type here while code is on your local machine don't create additional migration